xAPI-About.md

```diff
@@ -438,9 +434,7 @@ A Verb represents the "did" in "I did this".
 
 <a name="def-vocabulary"></a>
 
-__Vocabulary__: A list or collection of the terms that are used by a COP for labeling or categorizing information in a 
-particular domain. The use of a vocabulary ensures that everyone is using the same word to mean the same thing. For more 
-information on vocabularies, see the [xAPI Vocabulary Companion Specification](https://github.com/adlnet/companion-specification-for-xapi-vocabularies/blob/master/SUMMARY.md).
+__Vocabulary__: A vocabulary refers to the concepts and terms (e.g., Verbs, Activity Types) that are used by a COP for labeling or categorizing xAPI Profile information in a particular domain. Reusing vocabulary terms where appropriate ensures that everyone is using the same word to mean the same thing.
```

Comment:
What do you mean here? It looks like your reply was cut off.

Reply:
I'm proposing you do `e.g.` instead of `e.g.,`

To reword my comment "I don't think you need that first comma."

Reply:
Why e.g. instead of e.g., ? Is that a British vs. American English preference? 

Style guides do not agree on whether or not a comma should follow. I just searched through the other parts of the spec and it is very inconsistent. We seem to use both. Probably because so many people were contributing to it. So I think this should be resolved in another PR that makes this consistent throughout.

Reply:
#1067 just opened